### ISSUE TITLE
Dataset missing from Google Drive

### ISSUE THREAD

**Issue by Mikey-E** [chenhsuanlin/bundle-adjusting-NeRF#95]
Under the dataset section: [NeRF Google Drive](https://drive.google.com/drive/folders/128yBriW1IG_3NJ5Rp7APSTZsJqdJdfc1) seems to be empty.

Furthermore the gdown commands give:

"Cannot retrieve the public link of the file. You may need to change
 the permission to 'Anyone with the link', or have had many accesses.
 Check FAQ in https://github.com/wkentaro/gdown?tab=readme-ov-file#faq"

Can you please advise?

**Comment by Ahmed22347** [comment]
Yeah I have also noticed that , were you able to find the dataset.

**Comment by Ahmed22347** [comment]
I think it's the nerf dataset which can be found here:https://drive.google.com/drive/folders/1cK3UDIJqKAAm7zyrxRYVFJ0BRMgrwhh4

**Comment by Mikey-E** [comment]
I moved on to other things so I have no update wrt that. Good luck!
